testData/ontologies/flat-apptology.json

```diff
@@ -0,0 +1,18 @@
+{
+    "transform": [
+        {
+            "type": "fluid.transforms.value",
+            "inputPath": "http://registry\\.gpii\\.net/common/magnifierEnabled",
+            "outputPath": "http://registry\\.gpii\\.net/common/magnifierApplication"
+        }, {
+            "type": "fluid.transforms.value",
+            "inputPath": "http://registry\\.gpii\\.net/common/magnification",
+            "outputPath": "http://registry\\.gpii\\.net/common/magnifierApplication"
+        }
+    ],
+    "http://registry\\.gpii\\.net/common/screenReaderApplication": "http://registry\\.gpii\\.net/common/screenReaderTTSEnabled",
+    "http://registry\\.gpii\\.net/common/stickyKeysApplication": "http://registry\\.gpii\\.net/common/stickyKeys",
+    "http://registry\\.gpii\\.net/common/slowKeysApplication": "http://registry\\.gpii\\.net/common/slowKeysEnable",
```

Comment:
Yeah - that's pretty bad... The reason is that it's what it's called in our current ontologies (24751 and flat) along with debounceEnable. I agree that it should be changed, but probably as a separate pull

I've created a JIRA for it: https://issues.gpii.net/browse/GPII-1802